### PR TITLE
[WIP] A prettier site

### PR DESCRIPTION
**Pull request by joecritch**
This is only like an hour's work so far, as I've mostly been learning how Docusaurus works.

I might even delete this PR and re-submit again later, but I just wanted to check that this is the right approach with Docusaurus? It seems a lot of the layout isn't customisable with React components, and can only be changed with a custom stylesheet.

Although I might be limited to certain styles, I quite like Docusaurus so far. 

cc @vjeux

### CODE REVIEW

**Comment by vjeux** [comment]
cc @joelmarcey

**Comment by vjeux** [comment]
Could you post a screenshot of how this looks? The code looks good for me. Are there things you wanted to change but couldn't? @joelmarcey is actively working on open sourcing Docusaurus for real :)

**Comment by duailibe** [comment]
M

File: website/static/css/prettier.css [hunk 1]
```diff
@@ -0,0 +1,202 @@
+h1, h2, h3, h4, body {
```
Comment (lipis) [inline website/static/css/prettier.css]: run Prettier on this file :)

**Comment by joecritch** [comment]
@vjeux I'm closing this for now, as it didn't warrant a PR at this stage really. I'll re-open when it's looking healthier 👍

As for Docusaurus feedback, it'd be nice to be able to change the header, by decorating some base components, perhaps? I also feel like overriding the base styles could easily break with a version update, as they're targeting the internals of the base theme.

**Pull request closed without merging**